core/ui/public/i18n/en/translation.json

```
@@ -968,6 +968,7 @@
         "no_node_eligible_for_instance_cloning": "No node is eligible for instance cloning",
         "no_node_eligible_for_instance_migration": "No node is eligible for instance migration",
         "no_version_reason": {
+            "no_app_version": "No stable release of {app_name} is available in the \"{app_repository}\" repository",
```

```suggestion
            "no_app_version": "No stable release of {app_name} is available in the '{app_repository}' repository",
```
We usually use single quotes inside labels